src/core/reactor.cc

```diff
@@ -4136,15 +4158,39 @@ unsigned smp::adjust_max_networking_aio_io_control_blocks(unsigned network_iocbs
     auto requested_aio_network = network_iocbs * smp::count;
     auto requested_aio_other = (storage_iocbs + preempt_iocbs) * smp::count;
     auto requested_aio = requested_aio_network + requested_aio_other;
-    auto network_iocbs_old = network_iocbs;
+
+    seastar_logger.info("Intended AIO control block usage:");
+    seastar_logger.info("");
+    log_aiocbs(log_level::info, storage_iocbs, preempt_iocbs, network_iocbs);
+    seastar_logger.info("");
+    seastar_logger.info("Available AIO control blocks = aio-max-nr - aio-nr = {} - {} = {}", aio_max_nr, aio_nr, available_aio);
```

Comment:
the default  logging level is set to `info`, which ensures this message is always printed. i am wondering if we need this logging message if the number of aio cb is large enough. because i think, "no news is good news", which potentially reduces unnecessary verbosity.

so, maybe we can print it only when the number of aio cb is not sufficient and might hurt the network effienciency of the seastar application or it is not large enough to fulfill the minimum requirement?

Reply:
@tchaikov 

This patch (for seastar code) is also supposed to replace the following passage in scylladb's `docs/dev/docker-hub.md` (as of scylladb commit `ff52527c54e8`):
~~~
-Using multiple cores requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
-On many non-production systems, it will be equal to 65K. The formula
-to calculate the proper value is:
-
-    Available AIO on the system - (request AIO per-cpu * ncpus) =
-    aio_max_nr - aio_nr < (reactor::max_aio + detect_aio_poll + reactor_backend_aio::max_polls) * cpu_cores =
-    aio_max_nr - aio_nr < (1024 + 2 + 10000) * cpu_cores =
-    aio_max_nr - aio_nr < 11026 * cpu_cores
-
-    where
-
-    reactor::max_aio = max_aio_per_queue * max_queues,
-    max_aio_per_queue = 128,
-    max_queues = 8.
-
~~~

(See [here](https://github.com/scylladb/scylladb/issues/5981#issuecomment-2291900412) why -- it's just too stale, and seeing actual numbers is better than purely symbolic formulae.)

Given that the static documentation is going away, I believe that the basic info should be printed even if the resources prove sufficint. However, I do agree that seeing this message at every startup, with default log level configuration, becomes boring pretty fast. I suggest: downgrade the log level from info to verbose. Would that work for you?

Reply:
@lersek that'd work. thanks!

Reply:
correction: the log level just below `info` is `debug`, not `verbose` (I remembered `verbose` from another project, sorry).

Reply:
done in v2